fastlane/lib/fastlane/actions/hockey.rb

```diff
@@ -26,15 +20,6 @@ def self.connection(options)
           builder.response :json, content_type: /\bjson$/
           builder.use FaradayMiddleware::FollowRedirects
           builder.adapter :net_http
-          if ENV['SPACESHIP_DEBUG']
-            # for debugging only
-            # This enables tracking of networking requests using Charles Web Proxy
-            builder.proxy "https://127.0.0.1:8888"
```

Comment:
Do we need to set `builder.ssl[:verify_mode] = OpenSSL::SSL::VERIFY_NONE ` Here also? It feels like we are deleting something here that we might want?